tools/README.md

```diff
@@ -1,7 +1,21 @@
 # Boutiques Tools
 Tools to create and use boutiques descriptors.
 
-* validator.rb: a Ruby script to validate Boutiques descriptors.
-* pegasus-boutiques: a python API to generate Pegasus workflows from Boutiques JSON descriptors.
-* localExec.py: a Python script to test inputs on Boutiques JSON descriptors.
+Note: there is a Docker image ([boutiques/boutiques](https://hub.docker.com/r/boutiques/boutiques)) with the tools installed.
+
+1. `validator.rb`: a Ruby script to validate Boutiques descriptors. Requires Ruby >= 2.0.
+  * Installation:
+    * `gem install bundler`
+    * `bundle install`
+  * Usage:
+    * `validator.rb ../schema/descriptor.schema.json ./tool.json`
+  * Usage example with Docker image:
+    * `docker run --rm -v $PWD:/work -w /work boutiques/boutiques validator.rb /usr/local/boutiques/schema/descriptor.schema.json ./tool.json`
+
+2. `localExec.py`: a Python script to test inputs on Boutiques JSON descriptors.
+  * Usage example:
+    * Generates 4 random command lines: `docker run --rm -v $PWD:/work -w /work boutiques/boutiques localExec.py -n 4 -r ./tool.json`
+  * Check the help page (`-h` option) for more options and documentation.
```

Comment:
It is not necessary but if you wanted maybe we could add something like:
* Execute with an input file of parameters: 
`docker run --rm -v $PWD:/work -w /work boutiques/boutiques localExec.py -i input_param_file.json -e ./tool.json`

or

* Print a command line based on parameters from an input file: 
`docker run --rm -v $PWD:/work -w /work boutiques/boutiques localExec.py -i input_param_file.csv ./tool.json`